### DISCUSSION TITLE
boot slot detection failed for custom bootloader

### DISCUSSION THREAD

**Discussion by amorirReverse** [rauc/rauc#1352] [Q&A]
Hello,

I have a problem, I use rauc with a custom bootloader. When I run rauc status. 
The terminal shows me this: 
`maintenance@rescue2:~ $ rauc status
rauc-Message: 14:48:52.100: Config option 'statusfile=<path>/per-slot' unset, falling back to per-slot status
rauc-Message: 14:48:52.101: Using per-slot statusfile
Failed to determine slot states: Could not find any root device or rauc slot information in /proc/cmdline`

**Comment by jluebbe** [comment]
Please use English. You can edit your comment using the 'edit' action in the three-dot-menu on the top right.

**Comment by ejoerns** [comment]
@amorirReverse Do you pass any slot information via kernel commandline? Like `rauc.slot` or a `root=` rauc can identify as the root device? See https://rauc.readthedocs.io/en/latest/integration.html#identification-via-kernel-commandline

What kind of bootloader do you try to interface with?

Another option might be: https://github.com/rauc/rauc/pull/1215

**Reply by amorirReverse** [reply]
I'm coming back to you because I still have my error, my bone is a custom debian distribution with buildroot and my cmdline is : 
maintenance@rescue2:~ $ cat /proc/cmdline
coherent_pool=1M 8250.nr_uarts=0 snd_bcm2835.enable_headphones=0 bcm2708_fb.fbwidth=0 bcm2708_fb.fbheight=0 bcm2708_fb.fbswap=1 smsc95xx.macaddr=E4:5F:01:7F:15:0E vc_mem.mem_base=0x3ec00000 vc_mem.mem_size=0x40000000  console=tty1 logo.nologo panic=5 sdhci.debug_quirks2=4

**Comment by amorirReverse** [comment]
I can't use the detection via kernel commandline, because the cmbline.txt hasn't the information : root=....

**Reply by ejoerns** [reply]
So you had a look at look at #1215 which is exactly for this use case?

You also did not mention which bootloader you are trying to interface with.
